bowling/include/DataParser.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace {
+constexpr uint8_t strike = 10;
+constexpr uint8_t zeroPoints = 0;
+constexpr uint8_t addZeroPointsToVectorEndForThisSize = 25;
+constexpr char isStrike = 'X';
+constexpr char isMiss = '-';
+constexpr char isSpare = '/';
```

Comment:
`is*` in name suggest me that it is a function returning a bool. Don't use verbs for variables.
`convertCharToInt` below as well.

Reply:
Done